packages/cardano-services/src/InMemoryCache/InMemoryCache.ts

```diff
@@ -0,0 +1,87 @@
+import { CACHE_DURATION_DEFAULT } from './defaults';
+import NodeCache from 'node-cache';
+
+export type Key = string | number;
+export type DbSyncQuery<T> = () => Promise<T>;
+
+export class InMemoryCache {
+  #cache: NodeCache;
+  #ttlDefault: number;
+
+  constructor(cacheDurationInMins: number = CACHE_DURATION_DEFAULT) {
```

Comment:
```suggestion
  constructor(cacheTtlInMins: number = CACHE_TTL_DEFAULT) {
```
Let's rename all variables for consistency: cache duration -> cache TTL

Reply:
[00d4bba](https://github.com/input-output-hk/cardano-js-sdk/pull/276/commits/00d4bbae9dd517b46acd8abc7125bcff480b513c)